src/api/modelTypes.js

```diff
@@ -303,6 +303,10 @@ export type Stream = $ReadOnly<{|
   name: string,
   invite_only: boolean,
   is_announcement_only: boolean,
+  // TODO(server-2.1):
+  // is_web_public was added in Zulip version 2.1, absence implies the
+  // stream is not web public.
+  is_web_public?: boolean,
```

Comment:
nits on formatting TODO comments:

```suggestion
  // TODO(server-2.1): is_web_public was added in Zulip version 2.1;
  //   absence implies the stream is not web-public.
  is_web_public?: boolean,
```

(a) Including a summary of the thing-to-do on the same line with the TODO marker is helpful for making it a bit easier to see what it's about when doing a search like `git grep TODO.server-2.1`; instead of something like:
```
src/api/modelTypes.js
306:  // TODO(server-2.1):
```
one can get:
```
src/api/modelTypes.js
306:  // TODO(server-2.1): is_web_public was added in Zulip version 2.1;
```

(Often one needs to get a bit of context anyway, with something like `git grep -C2`. But then it's still helpful to have it kept compactly onto just a couple of lines.)

(b) Very small nit, but indenting subsequent lines of a TODO comment helps keep them visually grouped as part of the TODO, vs. being an independent comment. Some editors -- notably Android Studio, so this is relevant in our Kotlin/Java code -- have a distinct color to highlight TODO comments, and they rely on this same convention.

Oh and very small nit (c):
* https://github.com/zulip/zulip/pull/20976